### PR TITLE
SSI rigid example #271

### PR DESCRIPTION
**Pull request by somu15**
This example demonstrates a linear Soil Structure Interaction analysis assuming the structure has a concentrated mass on the top and is rigid.

### CODE REVIEW

File: doc/content/examples/example9.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+### Individual structural response
+
+The input file for simulating the individual structural response is presented below. The vertical beam representing the structure has a mass lumped at the top in addition to having a density. The horizontal beams connected to the rigid foundation are massless and they facilitate a rotational degree of freedom for the vertical beam at the node where it is intersecting with the foundation. The rigidity of both the vertical and the horizontal beams is achieved by setting properties such as area, moment of inertia, and Young's modulus to high values. The rigidity of the foundation is achieved by setting its Young's modulus to a high value. The Ormsby Wavelet was applied at the bottom of the rigid foundation and the response at the top of the structure was recorded. The figure below presents the 1% damped acceleration response spectra at the top of the structure. It is observed that at about 0.23s, the response spectra is peaking. This is the fundamental period of the structure. [!citet](BolisettiThesis) obtained a very similar value when modeling in LS-Dyna.
```
Comment (kyungtae79) [inline doc/content/examples/example9.md]: "The figure below presents the 1% damped acceleration response spectra at the top of the structure. It is observed that at about 0.23s, the response spectra is peaking. This is the fundamental period of the structure. [!citet](BolisettiThesis) obtained a very similar value when modeling in LS-Dyna." >>>> Figure 3 shows the 1% damped acceleration response spectra at the structure top with the fundament period of about 0.23 s. This value shows a good agreement with that in the LS-Dyna model by Bolisetti (2014).

File: doc/content/examples/example9.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+!listing examples/ex09/Soil_Response.i
+
+!media media/examples/ex09_Soil.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Soil
+       caption=Response spectra of the individual soil response under the Ormsby Wavelet.
+
+### Soil Structure Interaction response
+
+The input file for simulating the SSI response is presented below. The amplification ratio of the structural response spectra from the free-field soil is computed. Free-field soil is the soil region located far away from the structure, which in this case is one of the top end most nodes of the domain. This amplification ratio as a function of the period is presented below. Also presented below is the amplification ratio obtained by [!citet](BolisettiThesis) using LS-Dyna. It is seen that the variation of the amplification ratios with period is similar between MASTODON and LS-Dyna.
```
Comment (kyungtae79) [inline doc/content/examples/example9.md]: "This amplification ratio as a function of the period is presented below" >>> Figure 6 shows the amplification ratio, compared to that using LS-Dyna (Bolisetti 2014).

File: doc/content/examples/example9.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+!media media/examples/ex09_SSI_Valid.png
+       style=width:100% id=fig:SSI_Amp_Valid
+       caption=Amplification ratio computed using LS-Dyna by [!citet](BolisettiThesis).
+!col-end!
+
+!row-end!
+
+Additionally, the 1% acceleration response spectrum of the free-field soil response from SSI analysis is presented below. Along with it, the response spectrum obtained from the individual soil response is also plotted. It is seen that these to spectra are very similar.
+
+!media media/examples/ex09_SSI_FreeField.png
```
Comment (kyungtae79) [inline doc/content/examples/example9.md]: Figure 5??

File: doc/content/examples/example9.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+!col! small=12 medium=6 large=6
+!media media/examples/ex09_SSI_Valid.png
+       style=width:100% id=fig:SSI_Amp_Valid
+       caption=Amplification ratio computed using LS-Dyna by [!citet](BolisettiThesis).
+!col-end!
+
+!row-end!
+
+Additionally, the 1% acceleration response spectrum of the free-field soil response from SSI analysis is presented below. Along with it, the response spectrum obtained from the individual soil response is also plotted. It is seen that these to spectra are very similar.
```
Comment (kyungtae79) [inline doc/content/examples/example9.md]: Figure 8 shows the free-field soil acceleration response (1% damped) from the SSI analysis, compared to that obtained from the site response analysis.

File: doc/content/examples/example9.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+!listing examples/ex09/Soil_Response.i
+
+!media media/examples/ex09_Soil.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Soil
+       caption=Response spectra of the individual soil response under the Ormsby Wavelet.
+
+### Soil Structure Interaction analysis
+
+The input file for simulating the SSI response is presented below. The amplification ratio of the structural response spectra from the free-field soil is computed. Free-field soil is the soil region located far away from the structure, which in this case is one of the top end most nodes of the domain. [fig:SSI_Amp] presents the amplification ratio as a function of the period. [fig:SSI_Amp_Valid] presents the amplification ratio obtained by [!citet](BolisettiThesis) using LS-Dyna. It is seen that the variation of the amplification ratios with period is similar between MASTODON and LS-Dyna.
```
Comment (kyungtae79) [inline doc/content/examples/example9.md]: The amplification ratio of the structure (which elevation???) to surface free field is computed. For free-field response, the soil node is chosen far away from the structure to minimize any SSI effects. 

"[fig:SSI_Amp] presents the amplification ratio as a function of the period. [fig:SSI_Amp_Valid] presents the amplification ratio obtained by [!citet](BolisettiThesis) using LS-Dyna. It is seen that the variation of the amplification ratios with period is similar between MASTODON and LS-Dyna." >>>>
.....as a function of the period. This amplification ratio shows a good agreement with that using LS-Dyna (Bolisetti 20xx) as shown in Figure....

File: doc/content/examples/example9.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+[fig:SSI_FreeField] presents the free-field soil acceleration response (1% damped) from the SSI analysis, compared to that obtained from the site response analysis. It is seen that these to spectra are very similar.
+
+!media media/examples/ex09_SSI_FreeField.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:SSI_FreeField
+       caption=Free-field response spectra from the SSI analysis and the individual soil analysis.
+
+## Note on the computational effort required
+
+When a $dt$ of 0.002$s$ is used, for a 3$s$ Ormsby Wavelet, this problem has 1500 time steps. It is not recommended to run this problem on your local machine and it is better to use a supercomputer. For details on how to use an INL supercomputer, please see [FAQ](help/faq.md). On FALCON machine with 1 full node, this problem takes about 3-3.5 hours to run.
```
Comment (kyungtae79) [inline doc/content/examples/example9.md]: For this example, a INL supercomputer is used with dt of 0.002 sec for a total of 1500 time steps due to its expensive computational cost. For details on......

File: doc/content/examples/example9.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+!listing examples/ex09/Soil_Response.i
+
+!media media/examples/ex09_Soil.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Soil
+       caption=Acceleration response spectra of the soil subjected to the Ormsby Wavelet.
+
+### Soil Structure Interaction analysis
+
+The input file for simulating the SSI response is presented below. The amplification ratio of the structural response spectra at 10m height from the free-field soil is computed. Free-field soil is the soil region located far away from the structure, which in this case is one of the top end most nodes of the domain. [fig:SSI_Amp] presents the amplification ratio as a function of the period. This amplification ratio shows a good agreement with that using LS-Dyna computed by [!citet](BolisettiThesis) as presented in [fig:SSI_Amp_Valid].
```
Comment (cbolisetti) [inline doc/content/examples/example9.md]: I would argue that this is not a good agreement. 

I would actually suggest removing the comparison with the SDEE paper. The comparison shows that SASSI and LS-DYNA, which are completely different software have similar results, which are different from MASTODON. This is not good. Of course, the reason is that the model domain sizes are different, in addition to other differences.

File: doc/content/examples/example9.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+       style=width:80%;margin-right:0px;float:center;
+       id=fig:SSI_Amp
+       caption=Amplification ratio of the structural response spectra from the free-field soil in SSI analysis. This is computed using MASTODON.
+
+!media media/examples/ex09_SSI_Valid.png
+      style=width:80%;margin-right:0px;float:center;
+      id=fig:SSI_Amp_Valid
+      caption=Amplification ratio computed using LS-Dyna by [!citet](BolisettiThesis).
+
+[fig:SSI_FreeField] presents the free-field soil acceleration response (1% damped) from the SSI analysis, compared to that obtained from the site response analysis. It is seen that these to spectra are very similar.
```
Comment (cbolisetti) [inline doc/content/examples/example9.md]: Anything other than 'identical' here will raise questions. Please specify where the differences are coming from.

File: doc/content/examples/index.md [hunk 9]
```diff
@@ -13,6 +13,7 @@ A list of example simulations using MASTODON is provided below. These examples i
 - [Example 6c](examples/example6c.md): Force-controlled contact problem.
 - [Example 7](examples/example7.md): Demonstration of domain reduction method.
 - [Example 8](examples/example8.md): Seismic risk assessment of a generic nuclear facility using the MASTODON FTA Python module.
+- [Example 9](examples/example9.md): Soil Structure Interaction analysis of a rigid structure on linear soil without considering damping.
```
Comment (cbolisetti) [inline doc/content/examples/index.md]: Change case as suggested above

File: doc/content/examples/example9.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+!media media/examples/ex09_Ormsby.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:Ormsby
+       caption=Ormsby wavelet input ground motion used for the analysis.
+
+## Finite element mesh generation
+
+The three dimensional finite element mesh is generated using Cubit 15.5b. The procedure for generating the mesh is briefly described as follows:
+
+- The soil domain is first generated using the `Create volumes` option.
```
[the remark this answers is not in the document]
Comment (somu15) [reply doc/content/examples/example9.md]: Yes, I have done that now. Thank you for pointing.

**Comment by cbolisetti** [comment]
Recreating PR

**Pull request closed without merging**